src/components/Atoms/Picture/Picture.js

```diff
@@ -34,25 +37,65 @@ const Picture = ({
   height,
   objectFit,
   imageLow,
+  isBackgroundImage,
   ...rest
 }) => {
+  const document = typeof window !== 'undefined' ? window.document : null;
+  const [objFitState, setObjFitState] = useState(true);
+  let nonObjFitImage = null;
+
+  useEffect(() => {
+    // Once document is available, determine if this browser supports object-fit
+    if ('objectFit' in document.documentElement.style === false) { // TO-DO: SET BACK TO === FALSE!
+      setObjFitState(false);
+    }
+  }, [document]);
+
+  // Determine which image will be used for the nonObjectFit fallback
+  if (image) {
+    nonObjFitImage = image;
+  } else if (images) {
+    nonObjFitImage = images.substring(0, images.indexOf('?'));
+  }
+
   if (!images) {
     return (
-      <Wrapper height={height} width={width} {...rest}>
+      <Wrapper
+        height={height}
+        width={width}
+        image={image}
+        images={images}
+        isBackgroundImage={isBackgroundImage}
+        nonObjFitImage={nonObjFitImage}
+        objFitState={objFitState}
+        {...rest}
+      >
         <Image
           alt={alt}
           height={height}
           width={width}
           objectFit={objectFit}
           data-src={image}
-          className="lazyload"
+          className={`lazyload ${!objFitState && 'no-object-fit'}`}
```

Comment:
It's returning `false` as a class on image.

Reply:
I realised I don't even need that class anymore, so will remove it